app/javascript/packs/suspendUserModal.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
is this eslint disable needed? I removed it locally and eslint didn't shout at me 😂